README.md

```diff
@@ -58,6 +58,14 @@ Magika is available as `magika` on PyPI:
 $ pip install magika
 ```
 
+### Running on Docker
+
+```
+git clone https://github.com/google/magika
+cd python
+docker build -t magika .
+docker run -it -v $(path to mount):/magika --rm magika magika ./$filename
```

Comment:
Probably worth adding a workflow that publishes the image to ghcr.io

Reply:
Maybe separate pr?

Reply:
Yeah, agree